vpr/src/route/check_route.cpp

```diff
@@ -622,6 +626,18 @@ static void check_node_and_range(int inode, enum e_route_type route_type) {
     check_rr_node(inode, route_type, device_ctx);
 }
 
+//Checks that all non-configurable edges are in a legal configuration
+//This check is slow, so it has been moved out of check_route()
+static void check_all_non_configurable_edges() {
+    vtr::ScopedStartFinishTimer timer("Checking to ensure non-configurable edges are legal");
+    auto non_configurable_rr_sets = identify_non_configurable_rr_sets();
```

Comment:
I believe the non-config rr sets are now stored at a RR graph creation in the device_ctx, so we likely don't need to re-calculate it here.